src/piaf/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Article(models.Model):
+    CHOICES = [('pending', 'pending'), ('progress', 'progress'), ('completed', 'completed')]
+    name = models.CharField(max_length=100)
+    status = models.CharField(max_length=10, choices=CHOICES, default='pending')
+    user = models.ForeignKey(User, null=True, on_delete='null')
+    participated_at = models.DateField(auto_now=True)
+
+
+class Paragraph(models.Model):
+    article = models.ForeignKey(Article, on_delete='cascade', related_name='paragraphs')
+    text = models.TextField()
```

Comment:
This is an open question: do we need a "status" for the paragraphs as well ? It could be some computed property, but in terms of performance when we want to choose the next paragraph for the user, it could become interesting no ?

Reply:
Indeed. This will totally depend on whether or not one article can be processed by multiple users (one user per paragraph).

Reply:
In theory no. Only one user per article. I say _in theory_ because we may need to have multiple users in case one article goes from "in progress" to "pending" again ...

Reply:
But it could be fixed if we delete every contribution when the article goes from _progress_ to _pending_. We will then need to keep this in mind

Reply:
right. The idea for now is to reset the article when it's moving from _in progress_ to _pending_ again and therefore to clear the user.